src/CollectionSearchContainer.js

```diff
@@ -103,25 +136,41 @@ class CollectionSearchContainer extends React.Component {
     const qindex = e.target.value;
 
     this.props.mutator.query.update({ qindex });
-  }
+  };
 
-  passRecordsOut = records => {
+  passRecordsOut = (records) => {
     this.props.selectRecordsContainer(records);
-  }
+  };
 
   render() {
     const { onSelectRow, resources } = this.props;
-
+    const contentData = _.get(resources, 'metadataCollections.records', []);
+    const filterToCollections = _.get(resources, 'filterToCollections.records', []);
     if (this.collection) {
       this.collection.update(this.props, 'metadataCollections');
     }
 
+    // Here we filter collections is they are assigned or unassigned
+    // MAybe this block is better suited in CollectionsSearchContainer.js?
```

Comment:
Can you please remove the comments, that may be ambiguous in the future as well as the commented log statements?

Reply:
sure, but the first line with "// Here we filter collections is they are assigned or unassigned" may be quite useful. Could it stay?

Reply:
Yes, the 

> // Here we filter collections is they are assigned or unassigned" 

is quite useful. Please keep it!